packages/ui/src/app/pages/Proposals/ProposalPreview.tsx

```diff
@@ -153,6 +160,15 @@ export const ProposalPreview = () => {
                   <TextInlineMedium lighter>Time left:</TextInlineMedium>{' '}
                   <TextInlineMedium bold>{formatBlocksToDuration(blocksToProposalExecution)}</TextInlineMedium>{' '}
                   <TextInlineMedium lighter>({formatTokenValue(blocksToProposalExecution)} blocks)</TextInlineMedium>
+                  <TokenValueStat
+                    title={`Reward per ${rewardPeriod?.toString()} blocks`}
+                    value={rewardPeriod?.mul(opening.rewardPerBlock)}
+                  />
+                  <TokenValueStat
+                    title="Minimal stake"
+                    tooltipText="Minimal amount of tokens required to be staked for any applicant to such role."
+                    value={opening.stake}
```

Comment:
Hey @chrlschwb I don't understand how does adding these statistics relates to https://github.com/Joystream/pioneer/issues/3903 ?